internal/database/postgres.go

```diff
@@ -221,7 +221,9 @@ func (conn PostgresConn) UpdateGPUContext(host string, packet uplink.GPUInfo) er
 	return err
 }
 
-func (conn PostgresConn) Downsample(int_now time.Time) error {
+func (conn PostgresConn) Downsample(thresh time.Duration) error {
+	int_now := time.Now()
```

Comment:
NIT: This should be `intNow`. (And similarly, all the rest of the code should follow go's style)